<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Decision Tree</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>391</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Light</family>
      <pointsize>22</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(11, 34, 57);</string>
    </property>
    <property name="text">
     <string>Decision Tree Classifier</string>
    </property>
   </widget>
   <widget class="QListWidget" name="columns">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Columns:</string>
    </property>
   </widget>
   <widget class="QLabel" name="number_col">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Target : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dropout or Graduate&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train Size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="test_size">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test Data Size:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="test_size_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="test_data">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>0.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="train_size">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>20</width>
      <height>491</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Root Mean Sq. Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean Square Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mse">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="mae">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="rmse">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mean Absolute Error:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Accuracy Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>aa</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="report">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>270</y>
      <width>451</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Classification Report:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="predict_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Predict</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conf_mat_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>480</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Confusion Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="train">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>510</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Train</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_link">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Max_depth:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="max_depth">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Number of max_depth</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
